pages/dkp/kommander/2.1/release-notes/2.1.0/index.md

```diff
@@ -170,7 +170,35 @@ kubectl apply -f cert_manager_root-ca.yaml
 
 Once complete, continue to [attach your cluster][attach-cluster] to Kommander.
 
-You should expect to see that cert-manager will fail to deploy due to your existing `cert-manager` installation. This is expected and can be ignored.
+Your cert-manager will fail to deploy due to your existing `cert-manager` installation. This behavior is expected and can be ignored.
+
+### Kommander Cluster with custom SSL certificate
+
+When attaching a cluster, it is expected that the managed cluster was deploying apps as federated by the management cluster.
+If the management cluster was initialised using a custom SSL certificate, the managed cluster will fail cloning the manager's service repository. Check the status of the federated git repository resource to see the error:
```

Comment:
Do we do US English or UK English for the docs? @cmastr 

```suggestion
If the management cluster was initialized using a custom SSL certificate, the managed cluster will fail cloning the manager's service repository. Check the status of the federated git repository resource to see the error:
```

Reply:
US English.